### PR TITLE
Jl feature create contact

### PR DESCRIPTION
**Pull request by agalcalledjen**

### CODE REVIEW

File: client/js/components/Subheader/Subheader.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+export default props => (
+  <View {...props} style={[styles.subheaderContainer, props.style]}>
+    <View {...props} style={[styles.subheaderWrapper, props.style]}>
+      <CustomText {...props} style={[styles.subheader, props.style]}>
+        {props.children}
+      </CustomText>
+    </View>
+  </View>
+);
```
Comment (siddharthparmar7) [inline client/js/components/Subheader/Subheader.js:20]: proptype validations?

File: client/js/screens/Contact/Contact.js [hunk 2]
```diff
@@ -1,13 +1,74 @@
...
-import { View, Text } from 'react-native';
+import { View, Image, Linking } from 'react-native';
 import PropTypes from 'prop-types';
 
+import CustomText from '../../components/CustomText';
+import CustomIcon from '../../components/CustomIcon';
+import Subheader from '../../components/Subheader';
 import styles from './styles';
 
 const Contact = props => {
```
Comment (siddharthparmar7) [inline client/js/screens/Contact/Contact.js]: prop type validation?

File: client/js/components/CustomIcon/CustomIcon.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { Image, TouchableOpacity } from 'react-native';
+
+import styles from './styles';
+
+export default props => (
```
Comment (siddharthparmar7) [inline client/js/components/CustomIcon/CustomIcon.js:11]: add proptype validation.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Please make the requested changes. Thank you.

File: client/js/components/CustomIcon/CustomIcon.js (hunk 3 above)
Comment (agalcalledjen) [reply client/js/components/CustomIcon/CustomIcon.js:11]: Added.

File: client/js/screens/Contact/Contact.js (hunk 2 above)
Comment (agalcalledjen) [reply client/js/screens/Contact/Contact.js]: Added

File: client/js/components/Subheader/Subheader.js (hunk 1 above)
Comment (agalcalledjen) [reply client/js/components/Subheader/Subheader.js:20]: Added

**Review by siddharthparmar7 — APPROVED** [review]

**Pull request merged**